tmva/tmva/inc/TMVA/DNN/Minimizers.h

```diff
@@ -32,17 +31,17 @@ namespace DNN {
 *   independent implementation of the gradient descent minimization algorithm.
 *
 *   Provides Train(...) and TrainMomentum(...) functions that perform a complete
-*   training of a neural network. Thos are mainly used for testing since for
+*   training of a neural network. Those are mainly used for testing since for
 *   production a more fine grained control of the training process is desirable.
 *   This is provided by the Step(...), StepMomentum(...) and StepNesterov(...)
 *   functions that perform a single minimization step.
 *
 *   The main training characteristics are defined by the provided learning rate,
-*   the test interval and the convergence steps required for convergence. The
-*   test interval defines how often the error on the validation set is computed
-*   and is the values with which the step counter is increased each time
-*   the HasConverged() member function is called. A convergence step is defined as
-*   a step in which the test error is NOT less thatn 0.995 times the current
+*   the test interval, and the convergence steps required for convergence. The
+*   test interval defines how often the error on the validation set is computed,
+*   and the values with which the step counter is increased each time the
+*   HasConverged() member function is called. A convergence step is defined as
+*   a step in which the test error is NOT less than 0.999 times the current
```

Comment:
I preferred to change this to match the code rather than the other way around. The code has 0.999 in the test.